### PR TITLE
Setting the priority of Fault domain constraint to hard

### PR DESCRIPTION
**Pull request by milanmicic**
Updated paragraph describing the constraints priorities - removed Fault domain constrain as constraint considered as soft.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@milanmicic__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mmicic). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by PRmerger** [comment]
@milanmicic : Thanks for your contribution to the Azure documentation! The author, @masnider, has been notified to review your proposed change.

File: articles/service-fabric/service-fabric-cluster-resource-manager-management-integration.md [hunk 1]
```diff
@@ -110,9 +110,9 @@ ClusterManifest.xml
         </Section>
```
Comment (masnider) [inline articles/service-fabric/service-fabric-cluster-resource-manager-management-integration.md]: Are we missing the change to the actual default priority to the XML snippet above?

**Comment by masnider** [comment]
#sign-off

**Comment by masnider** [comment]
@milanmicic I wouldn't close it Milan, I think they still need to merge it.

**Comment by milanmicic** [comment]
@masnider Reopened, thanks

**Comment by azurecla** [comment]
Hi __@milanmicic__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mmicic). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by masnider** [comment]
@tysonn Hi Tyson, we took a look, this change is good.

**Comment by v-anpasi** [comment]
@tysonn Please merge.

**Comment by masnider** [comment]
Thanks @tysonn !

**Pull request merged**